usr/initiator.h

```diff
@@ -228,12 +228,14 @@ struct iscsi_ev_context {
 typedef struct queue_task {
 	iscsi_conn_t *conn;
 	iscsiadm_req_t req;
-	iscsiadm_rsp_t rsp;
 	int mgmt_ipc_fd;
 	unsigned int allocated : 1;
 	/* Newer request types include a
 	 * variable-length payload */
 	void *payload;
+  /* field 'rsp' with variable sized type 'iscsiadm_rsp_t' 
+   * must be at the end of a struct or class is a GNU extension */
+  iscsiadm_rsp_t rsp;
```

Comment:
This is indented incorrectly, as you can see from the diffs on github. If this field does indeed get moved, it needs to be indented correctly, and honestly I'd kind of like "req" to move with it, since they are paired somewhat (request, response).